doc/conf.py

```diff
@@ -34,6 +34,21 @@
 # are we running circle CI?
 CIRCLECI = 'CIRCLECI' in os.environ
 
+# shortcuts: subdirectories to not build.  Should be relative
+# to the doc directory.  Note that you cannot skip "users"
+# but can skip subdirectories...
+#
+# Note if any of the sphinx-galleries are skipped then they
+# will not be built using sphinx-gallery.
+skip_subdirs = []
+if False:
+    skip_subdirs = ['users/prev_whats_new/*',
+                    'gallery/*',
+                    'api/*',
+                    'tutorials/*',
+                    'plot_types/*',
+                    'devel/*']
```

Comment:
```suggestion
    skip_subdirs = [
        'users/prev_whats_new/*',
        'gallery/*',
        'api/*',
        'tutorials/*',
        'plot_types/*',
        'devel/*'
        ]
```
This format would allow me to lazily comment out the first or last subdir if that's the bit I _do_ want to build (though admittedly it's unlikely to be the first).